jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/thread/SerializedInvoker.java

```diff
@@ -45,8 +45,32 @@
     public Runnable offer(Runnable task)
     {
         if (task == null)
+        {
+            if (LOG.isDebugEnabled())
+                LOG.debug("Offering task null, skipping it in {}", this);
```

Comment:
Necessary?

Reply:
Well, since `offer(Runnable...)` is logging `Offering 3 tasks in SerializedInvoker@abc123`, this makes sure we're logging 3 times `Offering ...` even if some tasks are null otherwise we may log `Offering ...` less times which may be confusing?